### DISCUSSION TITLE
i your help for 2m dogecoin

### DISCUSSION THREAD

**Discussion by abosaleh82** [dogecoin/dogecoin#3176] [Help needed] [answered]
I have dogecoin wallet and i lost my password when i contact dogecoin chain the say they don't keep it as policy , but after times of sending a lot of mails they give GitHub link and say there you will find a solution , who can    ???

with my best regards,
  abo-saleh

**Comment by etherx-dev** [comment, marked as answer]
https://linkedin.com/in/in- I can help

**Comment by SandroMarques46** [comment]
Nothing can be done. You should have gotten the private keys backup. The wallet-id and password for dogechain are worthless if you don't remember neither of them.

**Reply by etherx-dev** [reply]
With the right assessment snd data forensic Understanding ( notice understanding, not motivation) anything is possible. 

There are scenarios that over the years i end up resolving, but i still cant recover everything i lost.  

I do find it easy yo do what i had yo learn in years in hours if not minutes now though 😀

To the OP: how sure ate you that you had this much coins, do you remember sending crypto from an exchange? Find the exchange and the txid so we can start trom there.


I would advise on archiving data that could give any clues to resolving your loss.


Also like i said, do your research and find me on linkedin leme help. I wont be asking for anything remotely linked to your wallet apart from proof of your sanity

**Reply by SandroMarques46** [reply]
Why do you want to go to linked in when you can solve the problem here , **publicly** ? Strange...

**Reply by abosaleh82** [reply]
how that ,, and i have my world who helping to bring or transferred my coin from dogecoin chain to my wallet in other place ,

lets start to solve it

**Reply by etherx-dev** [reply]
> Why do you want to go to linked in when you can solve the problem here , **publicly** ? Strange...

@etherx-dev

**Reply by etherx-dev** [reply]
> Why do you want to go to linked in when you can solve the problem here , **publicly** ? Strange...

Because anyone can open a github account, i need to know who this is and so does the OP.


- [ ] Go ahead and share your solution, dr strange.

**Comment by SandroMarques46** [comment]
The positive side is that **_I think_** you can "brute force" (try as many passwords as you want) since dogechain doesn't block that so if you remember a part of the password just keep trying different ones.

**Comment by abosaleh82** [comment]
hello sir,

What is your success percentage>>?? to get back my coin

thanks for you
